File: ViewletCreatorIOS/ViewletCreator/Classes/utility/ViewletColorLookup.swift
//
//  ViewletColorLookup.swift
//  Viewlet creator Pod
//
//  Library utility: protocol for color lookup
//  Integrates with ViewletConvUtil to look up custom color references
//

import UIKit

public protocol ViewletColorLookup {
    
    func getColor(refId: String) -> UIColor?
    
}

File: ViewletCreatorAndroid/ViewletCreatorLib/src/main/java/com/crescentflare/viewletcreator/utility/ViewletColorLookup.java
package com.crescentflare.viewletcreator.utility;

import org.jetbrains.annotations.Nullable;

/**
 * Viewlet creator utility: interface for color lookup
 * Integrates with ViewletMapUtil to look up custom color references
 */
public interface ViewletColorLookup
{
    @Nullable
    Integer getColor(String refId);
}
